src/test/rules/helper.ts

```diff
@@ -24,13 +25,16 @@ export function testScript(rule: string, scriptText: string, config: Object): bo
 /**
  * @deprecated Use ruleTester
  */
-export function makeTest(rule: string, scripts: Array<string>, expected: boolean, config?: { rules: {} }) {
+export function makeTest(rule: string, scripts: Array<string>, expected: boolean, config?: Lint.Configuration.IConfigurationFile) {
```

Comment:
consider adding a new helper that converts your rule config to `Lint.Configuration.IConfigurationFile` instead of having the boilerplate code in all tests.